index.html

```diff
@@ -48,6 +48,109 @@ <h1 class="logo-text">Pascal</h1>
         </div>
     </section>
 
+    <section class="work">
+
+        <div class="card">
+            <div class="card-header">
+            </div>
+            <h2 class="card-titles">TONIC</h2>
+
+
```

Comment:
## OPTIONAL
- [ ] Please avoid more than two(2) empty lines. Also, only create an empty line for purposes.